django/otto/celery.py

```diff
@@ -48,6 +48,11 @@
     #     "task": "otto.tasks.delete_unused_libraries",
     #     "schedule": crontab(hour=3, minute=0),
     # },
+    # Delete unused libraries every day at 3 am UTC
+    "delete-unused-libraries-every-morning": {
+        "task": "otto.task.delete_old_libraries",
```

Comment:
I would call this "delete_unused_libraries" not "delete_old_libraries" throughout, since it is now based on access, not when the library was created.